.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Backpack CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: nvm install
+      shell: bash -l {0}
+      run: nvm install
+    
+    - name: nvm use
+      shell: bash -l {0}
+      run: nvm use
+    
+    - name: Install deps
+      run: npm ci
+    
+    - name: Build Backpack
+      run: npm run build
+
+    - name: Check package-lock
+      run: ./scripts/check-pristine-state package-lock.json
+    
+    - name: Test
+      run: npm test
+    
+    - name: Build Storybook
+      if: github.ref == 'refs/heads/master'
+      run: npm run storybook:dist
+    
+    - name: Build Sass docs
+      if: github.ref == 'refs/heads/master'
+      run: npm run sassdoc
+
+    - name: Deploy sassdoc
+      uses: peaceiris/actions-gh-pages@v3
+      if: github.ref == 'refs/heads/master'
+      with:
+        personal_token: ${{ secrets.DEPLOY_TOKEN }}
+        publish_dir: dist-sassdoc/
+        keep_files: true
+        external_repository: backpack/sassdoc
+        publish_branch: master
+    
+    - name: Deploy storybook
+      uses: peaceiris/actions-gh-pages@v3
+      if: github.ref == 'refs/heads/master'
+      with:
+        personal_token: ${{ secrets.DEPLOY_TOKEN }}
+        publish_dir: dist-storybook/
+        keep_files: true
+        external_repository: backpack/storybook
+        publish_branch: master
```

Comment:
Can we have a new line at the end of the file?

Reply:
Merged before I saw this, will add something to master now :)